src/AutoBuild.jl

```diff
@@ -603,6 +603,32 @@ function get_meta_json(
     return dict
 end
 
+function compose_debug_prompt(sandbox_dir, project_dir)
+    log_files = String[]
+    for (root, dirs, files) in walkdir(joinpath(sandbox_dir, "srcdir"))
+        for file in files
+            if endswith(file, ".log")
+                push!(log_files, replace(joinpath(root, file), "$project_dir/" => ""))
```

Comment:
Why using two arguments?  I'd simply do
```suggestion
function compose_debug_prompt(workspace)
    log_files = String[]
    for (root, dirs, files) in walkdir(joinpath(workspace, "srcdir"))
        for file in files
            if endswith(file, ".log")
                push!(log_files, replace(joinpath(root, file), workspace => "\${WORKSPACE}"))
```